articles/stay-on-course-with-your-security-compliance-goals.md

```diff
@@ -0,0 +1,55 @@
+
+Let’s look at the Payment Card Industry Data Security Standard or PCI DSS. This is a set of standards that aims to protect debit and credit card transactions against data theft and fraud. This is pretty important if your organization plans to accept card payments. But how many of your teams are in the position to collect payments — or even interact with customers?
+
+There may be processes that apply to your entire company, like using multi-factor authentication. But this isn’t always the case. Knowing which compliance standards apply to which teams will help lighten the load across your organization.
+
+## Delegate to team leads
+
+Ultimately, your Head of Security or Chief Compliance Officer is responsible for ensuring compliance, depending on the structure of your organization. Asking them to check the progress of each team and individual, in addition to their other responsibilities, isn’t exactly efficient (or fair).
+
+Your company could hire more security specialists. Some organizations have thousands on staff, but we know this isn’t always a realistic option. The good news is that you don’t need more security people — just more people with security skills.
+
+Team leads are in the perfect position to help you measure compliance progress. They know which hardware needs maintenance. They know which software to update, and they’ll be the first to know of any employees who are onboarding or offboarding.
+
+## Fleet makes measuring compliance easy
+
+Your team leads don’t need a crash course in security and compliance to measure compliance progress. With Fleet, you can create [policies](https://fleetdm.com/securing/what-are-fleet-policies) that ask questions about your devices — questions you can customize to meet your compliance goals. You can then group these custom queries with [teams](https://fleetdm.com/docs/using-fleet/teams), eliminating unnecessary processes and lowering the level of effort for your organization.
+
+Fleet also lets you [assign users different levels of access](https://fleetdm.com/docs/using-fleet/permissions): Observer, Maintainer, and Admin. So, your Head of Security can create policies and assign them to different teams — preparing team leads to run queries for up-to-the-minute data on devices. Though that might not be necessary. Fleet policies automatically refresh every hour.
+
+Fleet policies, teams, and permissions empower employees at every level of your organization to share the responsibility of measuring compliance. With more hands on deck, reaching your goals will be a breeze.
+
+
+<meta name="category" value="product">
+<meta name="authorFullName" value="Chris McGillicuddy">
+<meta name="authorGitHubUsername" value="fleetdm">
+<meta name="publishedOn" value="2022-06-14">
+<meta name="articleTitle" value="Stay on course with your security compliance goals">
+<meta name="articleImageUrl" value="../website/assets/images/articles/fleet-4.15.0-cover-1600x900@2x.jpg">
```

Comment:
Can you remove this meta tag or add an image for this article and update the value?